resources/plugins/QuantME-CamundaPlugin/quantme/modeling/QuantMEFactory.js

```diff
@@ -33,6 +33,12 @@ export default class QuantMEFactory extends BpmnFactory {
       var prefix = (element.$type || '').replace(/^[^:]*:/g, '') + '_';
       element.id = this._model.ids.nextPrefixed(prefix, element);
     }
+
+    // setting default for selectlist
+    if (element.$type === 'quantme:ReadoutErrorMitigationTask') {
```

Comment:
Is available as a constant here: https://github.com/UST-QuAntiL/QuantME-TransformationFramework/blob/develop/client/src/app/quantme/Constants.js#L18
Please use that instead